lib/jekyll-commonmark.rb

```diff
@@ -30,14 +30,16 @@ def initialize(config)
             true
           end
         end
+
+        options_set = Set.new(options).freeze
+        @parse_options = (options_set & CommonMarker::Config::Parse.keys).to_a
+        @render_options = (options_set & CommonMarker::Config::Render.keys).to_a
+        @parse_options = :DEFAULT if @parse_options.empty?
+        @render_options = :DEFAULT if @render_options.empty?
```

Comment:
shouldn't it be `[:DEFAULT]` if you need the options to be an Array..? or is `:DEFAULT` already mapped to an array?

Reply:
It's goofy, but it can be a symbol _or_ an array. Passing a single symbol is faster if there is only one.

https://github.com/gjtorikian/commonmarker/blob/adf83c5de106a6b9d9e2c9b8a14dbad962a3001b/lib/commonmarker/config.rb#L28-L34